src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,59 @@
+$exec-ed-error-heading-font-size: $font-size-base * 3.34;
+
+
+.exec-ed-error-background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 78px;
+    z-index: -1;
+}
+
+.executive-education-2u-error-heading {
+    line-height: $headings-line-height;
+
+    .executive-education-2u-error-heading-red {
+        font-size: $exec-ed-error-heading-font-size;
+        font-weight: $font-weight-bold;
+        color: $brand-500;
+        margin-right: 10px;
```

Comment:
nit: Is there a spacing token to use instead of hardcoding a pixel value? Would recommend only using spacing values that exist in our design system (e.g., `mr-2` (8px) or `mr-2.5` (12px)).